hmac-protected/hmac-protected/handler.py

```diff
@@ -0,0 +1,30 @@
+import os, hmac, hashlib
+
+def validateHMAC(message,secret,hash):
+
+    # The hash starts with the encryption method "sha1=" we need to remove it
+    cleanHash = hash[5:]
+    encodedSecret = secret.encode()
+    encodedMessage = message.encode()
+
+    # Encrypt message with secret
+    expectedMac = hmac.new(encodedSecret,encodedMessage,hashlib.sha1)
+    hmacDigest = expectedMac.hexdigest()
+
+    # Compare the received hash with created hash
+    if hmacDigest == cleanHash:
+        return True
+    return False
+
+def handle(req):
+    # We receive the encrypted message in form of a header
+    encryptedMessage = os.getenv("Http_Hmac")
```

Comment:
Good that you've tried to follow the nomenclature that Alex uses in the go package.

Can we use messageMAC rather than encryptedMessage. As well as removing 'encrypted' I think this makes the code clearer as we are comparing messageMAC with expectedMAC